pkg/tfbridge/schema.go

```diff
@@ -341,6 +341,24 @@ func MakeTerraformInputs(
 	return makeTerraformInputsWithOptions(ctx, instance, config, olds, news, tfs, ps, makeTerraformInputsOptions{})
 }
 
+// Converts a single Pulumi property value into a plain go value suitable for use by Terraform.
+// This does not apply any defaults or other transformations.
+func makeSingleTerraformInput(
```

Comment:
Nit: go prefers function descriptions to name the function.
```suggestion
// makeSingleTerraformInput converts a single Pulumi property value into a plain go value suitable for use by Terraform.
//
// makeSingleTerraformInput does not apply any defaults or other transformations.
func makeSingleTerraformInput(
```